grails-app/migrations/0.9.x/changelog-2025-02-21-0000-remove-cycle-count-debit-transaction-type.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="ewaterman" id="210220250000-0">
```

Comment:
Let's not do this here. Even if PIH has never created a debit adjustment it's possible there are existing transactions in other system so this is going to lead to some support headaches.

Reply:
Nevermind, I just realized this was added erroneously. 

If it's possible, I would rather erase the earlier database migration (and manually remove the data from all "dev" databases). Since this never made it to production, that would be better than adding and removing.